proto/schema/v1/schema.proto

```diff
@@ -0,0 +1,41 @@
+syntax = "proto3";
+
+package schema.v1;
+
+service SchemaService {
+  rpc Register(RegisterRequest) returns (RegisterResponse);
+  rpc Fetch(FetchRequest) returns (FetchResponse);
+}
+
+message Schema {
+  enum Type {
+    TYPE_UNSPECIFIED = 0;
+    TYPE_AVRO = 1;
+  }
+
+  // The schema's globally unique ID
+  string id = 1;
+  // The schema's name (globally unique)
+  string name = 2;
+  int32 version = 3;
+  Type type = 4;
+  // The schema contents
+  bytes bytes = 5;
+}
+
+message RegisterRequest {
+  // The schema name. It needs to be unique in the scope of a connector.
+  string name = 1;
+  Schema.Type type = 2;
+  bytes bytes = 3;
+}
+message RegisterResponse {
+  string id = 1;
```

Comment:
How about we return the whole created schema back? When working with schema registries in the past I found it frustrating that I had to create a schema and then right after fetch the schema if I wanted to know for example the version that was assigned to the schema.

Reply:
I was thinking about that too, but wasn't sure if there was a use case... In a source connector's context, when would it want to know the created schema's version? On the other hand, it doesn't hurt having it, and we don't loose anything.:)

Reply:
If we add it, we'll never need it. If we don't, we'll need it as soon as we cut a release and changing this would create a breaking change 😄

Reply:
Exactly!